src/Dialog/Dialog.js

```diff
@@ -463,6 +478,10 @@ class Dialog extends Component {
     autoScrollBodyContent: false,
     modal: false,
     repositionOnUpdate: true,
+    paperProps: {
```

Comment:
Why? The `Paper` is receive `{rounded: false, zDepth: 1}` as default props values.